src/components/BirdBackground.tsx

```
@@ -208,41 +264,33 @@ const BirdBackground: React.FC = () => {
-          console.log('BirdBackground: PixiJS app instance from ref destroyed successfully.');
         } catch (e) {
-          console.error('BirdBackground: Error during PixiJS app (from ref) stop/destruction:', e);
+          /* ignore */
         }
         pixiAppRef.current = null; 
       }
 
       if (localAppInstance && localAppInstance !== appInRefToDestroy && typeof localAppInstance.destroy === 'function') {
-          console.warn('BirdBackground Cleanup: localAppInstance was not the one in ref (or ref was null), attempting to destroy it as an orphaned instance.');
           try { 
             if (localAppInstance.ticker && localAppInstance.ticker.started) { 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)